lib/govuk_index/presenters/elasticsearch_presenter.rb

```diff
@@ -40,7 +40,9 @@ def document
         specialist_sectors:                 expanded_links.specialist_sectors,
         taxons:                             expanded_links.taxons,
         topic_content_ids:                  expanded_links.topic_content_ids,
-        indexable_content:                  indexable_content.indexable_content
+        indexable_content:                  indexable_content.indexable_content,
+        licence_identifier:                 licences.identifier,
+        licence_short_description:          licences.short_description,
```

Comment:
I would like to combine the indexable_content and licences presenters into a single `DetailsPresenter` as we don't want to end up with 100's of presenters, instead I think we should aim to have as few as we can get away with and still have each dealing with related data.

Ignore if you disagree